### PR TITLE
Replaced most references to GitLab with GitHub. 

### PR DESCRIPTION
**Pull request by lemonsaurus**
There's still GitLab stuff in the wiki because that needs a proper rewrite, and there's still GitLab stuff in the code jam team pages, because they would break if I changed them.

### CODE REVIEW

**Review by jchristgit — DISMISSED** [review]

**Review by sco1 — DISMISSED** [review]

File: templates/main/base.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
                     <p>
                         This website uses <a href="https://python.org">Python</a> and
                         <a href="http://flask.pocoo.org/">Flask</a>, and was developed collaboratively on
-                        <a href="https://gitlab.com/python-discord/projects/site">GitLab</a>.
+                        <a href="https://github.com/python-discord/projects/site">GitLab</a>.
```
Comment (jb3) [inline templates/main/base.html]: The text should probably be changed here.

File: pysite/views/main/redirects/gitlab.py [hunk 2]
```diff
@@ -4,5 +4,5 @@
 class GitLabView(RedirectView):
     path = "/gitlab"
     name = "gitlab"
-    page = "https://gitlab.com/python-discord/"
+    page = "https://github.com/python-discord/"
```
Comment (jb3) [inline pysite/views/main/redirects/gitlab.py:7]: This view intentionally points to GitLab, we already have another view which redirects to GitHub.

File: package.json [hunk 3]
```diff
@@ -16,10 +16,10 @@
   "devDependencies": {},
   "repository": {
     "type": "git",
-    "url": "git@gitlab.com:discord-python/projects/site.git"
+    "url": "git@github.com:python-discord/projects/site.git"
   },
   "homepage": "https://pythondiscord.com",
-  "bugs": "https://gitlab.com/python-discord/projects/site/issues",
+  "bugs": "https://github.com/python-discord/projects/site/issues",
```
Comment (jb3) [inline package.json]: This URL is incorrect.

File: package.json [hunk 4]
```diff
@@ -16,10 +16,10 @@
   "devDependencies": {},
   "repository": {
     "type": "git",
-    "url": "git@gitlab.com:discord-python/projects/site.git"
+    "url": "git@github.com:python-discord/projects/site.git"
```
Comment (jb3) [inline package.json]: This URL is incorrect.

File: templates/wiki/base.html [hunk 5]
```diff
@@ -201,7 +201,7 @@
                     <p>
                         This website uses <a href="https://python.org">Python</a> and
                         <a href="http://flask.pocoo.org/">Flask</a>, and was developed collaboratively on
-                        <a href="https://gitlab.com/python-discord/projects/site">GitLab</a>.
+                        <a href="https://github.com/python-discord/projects/site">GitLab</a>.
```
Comment (jb3) [inline templates/wiki/base.html]: This URL is incorrect, not only does it point to a projects group on GitHub but it still reads GitLab.

File: templates/main/base.html (hunk 1 above)
Comment (jb3) [reply templates/main/base.html]: This URL is also incorrect, it is https://github.com/python-discord/site not https://github.com/python-discord/projects/site

File: pysite/views/error_handlers/http_5xx.py [hunk 6]
```diff
@@ -36,6 +36,6 @@ def error(self, error: HTTPException):
             "errors/error.html", code=error.code, req=request, error_title=error_desc,
             error_message="An error occurred while processing this request, please try "
                           "again later. If you believe we have made a mistake, please "
-                          "<a href='https://gitlab.com/python-discord/projects/site/issues'>file an issue on our"
-                          " GitLab</a>."
+                          "<a href='https://github.com/python-discord/projects/site/issues'>file an issue on our"
```
Comment (jb3) [inline pysite/views/error_handlers/http_5xx.py]: Again, wrong URL, needs to have that projects removed.

File: pysite/views/error_handlers/http_4xx.py [hunk 7]
```diff
@@ -26,6 +26,6 @@ def error(self, error: HTTPException):
         return self.render(
             "errors/error.html", code=error.code, req=request, error_title=error_desc,
             error_message=f"{error_desc} If you believe we have made a mistake, please "
-                          "<a href='https://gitlab.com/python-discord/projects/site/issues'>"
-                          "open an issue on our GitLab</a>."
+                          "<a href='https://github.com/python-discord/projects/site/issues'>"
```
Comment (jb3) [inline pysite/views/error_handlers/http_4xx.py]: Incorrect URL, needs to have the projects group removed.

File: templates/wiki/base.html (hunk 5 above)
Comment (jb3) [reply templates/wiki/base.html]:
```suggestion
                        <a href="https://github.com/python-discord/site">GitHub</a>.
```

File: templates/main/base.html (hunk 1 above)
Comment (jb3) [reply templates/main/base.html]:
```suggestion
                        <a href="https://github.com/python-discord/site">GitHub</a>.
```

File: pysite/views/error_handlers/http_5xx.py (hunk 6 above)
Comment (jb3) [reply pysite/views/error_handlers/http_5xx.py]:
```suggestion
                          "<a href='https://github.com/python-discord/site/issues'>file an issue on our"
```

File: pysite/views/error_handlers/http_4xx.py (hunk 7 above)
Comment (jb3) [reply pysite/views/error_handlers/http_4xx.py]:
```suggestion
                          "<a href='https://github.com/python-discord/site/issues'>"
```

File: package.json (hunk 3 above)
Comment (jb3) [reply package.json]:
```suggestion
  "bugs": "https://github.com/python-discord/site/issues",
```

File: package.json (hunk 4 above)
Comment (jb3) [reply package.json]:
```suggestion
    "url": "git@github.com:python-discord/site.git"
```

**Review by jb3 — CHANGES_REQUESTED** [review]
There are still quite a few references to GitLab in the code that are not resolved with this PR, I've got the other branch that I made when I didn't realize that this was a thing which covers a few more things that aren't covered in here.

Also most of the references here have just basically `s/github/gitlab` but that doesn't work for most cases as GitLab had a projects group.

File: pysite/views/main/redirects/gitlab.py (hunk 2 above)
Comment (lemonsaurus) [reply pysite/views/main/redirects/gitlab.py:7]: but.. the GitHub view was pointed at GitLab as well?

Comment (lemonsaurus) [reply pysite/views/main/redirects/gitlab.py:7]: The point being - we should just redirect to github no matter what. there's no point in directing to a dead gitlab page.

**Review by jb3 — DISMISSED** [review]

File: pysite/views/main/redirects/gitlab.py (hunk 2 above)
Comment (jchristgit) [reply pysite/views/main/redirects/gitlab.py:7]: Wouldn't it make sense to also change the path then?

File: templates/main/info/resources.html [hunk 8]
```diff
@@ -83,7 +83,7 @@ <h2 class="uk-heading-divider hover-title" id="{{ category_name.replace(" ", "-"
                                 <div class="resource-title">
                                     <strong>{{ item }}</strong> <br />
                                     <div class="uk-button-group">
-                                        {% for url in data.urls %}
+            g                           {% for url in data.urls %}
```
Comment (jchristgit) [inline templates/main/info/resources.html]: This seems unintentional.

Comment (jchristgit) [reply templates/main/info/resources.html]:
```suggestion
                                        {% for url in data.urls %}
```

**Review by jchristgit — CHANGES_REQUESTED** [review]
Space rollercoaster

**Review by jchristgit — DISMISSED** [review]

**Review by jchristgit — DISMISSED** [review]
:shipit::shipit::shipit::shipit::shipit::shipit::shipit:

File: pysite/views/main/redirects/gitlab.py (hunk 2 above)
Comment (jb3) [reply pysite/views/main/redirects/gitlab.py:7]: @jchristgit why? we already have the GitHubView

Comment (jchristgit) [reply pysite/views/main/redirects/gitlab.py:7]: I meant, change the path as in remove it completely

**Review by scragly — APPROVED** [review]

**Review by sco1 — APPROVED** [review]
Per discussion in #dev-contrib yesterday seems like this has covered everything non-jam related

**Pull request merged**